app/Controllers/Http/ProfilesController.ts

```diff
@@ -0,0 +1,81 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import Database from '@ioc:Adonis/Lucid/Database'
+import UserProfile from 'App/Models/UserProfile'
+import UpdateProfileValidator from 'App/validators/UpdateProfileValidator'
+import Utils from 'App/utils/utils'
+import { Gender } from 'Contracts/Enums'
+import {DateTime} from 'luxon'
+
+export default class ProfilesController {
+  public async createProfile({}: HttpContextContract) {
+    return 200
+  }
+
+  public async getProfile({ auth, response, request }: HttpContextContract) {
+    try {
+    const email = auth.user?.email
+    const user = await Database.from('user as u')
+      .innerJoin('user_profiles as p', 'u.id', 'p.user_id')
+      .where({ email })
+      .select(['email', 'mobile', 'gender', 'dob'])
+      .first()
```

Comment:
Use relation here, You can use preload method to load the userProfile. Something below

```ts
const user = await auth.use('api').user

const data = await User.query().where('id', user.id).preload('profile').first();

            return {
                'email': data?.email,
                'name': data?.profile?.name,
                'gender': data?.profile?.gender,
                'dob': data?.profile?.dob
            }
```

Reply:
done